modules/sdk-core/src/account-lib/mpc/util.ts

```diff
@@ -31,6 +31,10 @@ export function bigIntFromBufferBE(buf: Buffer): bigint {
   return BigInt('0x' + buf.toString('hex'));
 }
 
+/**
+ * @deprecated Use bigIntFromU8ABE from sdk-lib-mpc instead
+ * @param buf
+ */
 export function bigIntFromU8ABE(buf: Uint8Array): bigint {
```

Comment:
Same, lets use it from sdk-lib-mpc and export here along side the deprecation notice so that any changes made to this function in the sdk-lib-mpc would also be reflected here

Reply:
Fixed.